.github/workflows/repair-chat-capability-finalize-v2.yml

```
@@ -0,0 +1,395 @@
+
+      - name: Finalize transport and ordinary-role classification
+        shell: bash
+        run: |
+          set -euo pipefail
+          python - <<'PY'
+          from pathlib import Path
+          import re
+
+          CHAT_CAPABILITY = '''"""Classify chat transport compatibility and ordinary agent-role eligibility.
+
+Provider catalogs mix endpoint-only models with models served through an
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)